services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java

```diff
@@ -472,7 +472,9 @@ protected void createCommonAttributes(Theme theme, Locale locale, Properties mes
 
             attributes.put("url", new UrlBean(realm, theme, baseUri, this.actionUri));
             attributes.put("requiredActionUrl", new RequiredActionUrlFormatterMethod(realm, baseUri));
-            attributes.put("auth", new AuthenticationContextBean(context, page));
+            if (context != null) {
```

Comment:
I don't think we can remove the bean but make sure the `AuthenticationContextBean.getAttemptedUsername` is checking for a null context.

The reason is that other methods are making this check and they indicate the bean can be used when the context is not available.

Reply:
@pedroigor ok, I'll change it.